src/Request/Read/Transaction.php

```diff
@@ -1,8 +1,6 @@
 <?php
 namespace PhpTwinfield\Request\Read;
 
-use PhpTwinfield\Office;
```

Comment:
Can you put all the Office typehints back?

Reply:
See above